src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -6270,10 +6270,10 @@ instruct cmovI_regUCF2_ne_ndd(cmpOpUCF2 cop, rFlagsRegUCF cr, rRegI dst, rRegI s
 
   ins_cost(200);
   format %{ "ecmovpl  $dst, $src1, $src2\n\t"
-            "ecmovnel $dst, $src1, $src2" %}
+            "cmovnel  $dst, $src2" %}
```

Comment:
You need `effect(TEMP dst)` for these nodes, too.

Reply:
Thank you! Please see the `effect(TEMP dst)` added for all the match rules.